tasks/run-matlab-command/v1/task.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "28fdff80-51b4-4b6e-83e1-cfcf3f3b25a6",
+    "name": "RunMATLABCommand",
+    "friendlyName": "Run MATLAB Command",
+    "description": "Execute a MATLAB script, function, or statement. MATLAB exits with exit code 0 if the specified script, function, or statement executes successfully without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the build to fail. You can use the assert or error functions in the command to ensure that builds fail when necessary. When you use this task, all of the required files must be on the MATLAB search path.",
+    "helpMarkDown": "",
+    "category": "Build",
+    "author": "The MathWorks, Inc.",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "inputs": [
+        {
+            "name": "command",
+            "type": "string",
+            "label": "Command",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "Script, function, or statement to execute. If the value of `command` is the name of a MATLAB script or function, do not specify the file extension. If you specify more than one MATLAB command, use a comma or semicolon to separate the commands."
+        },
+        {
+            "name": "startupOptions",
+            "type": "string",
+            "label": "Startup Options",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Startup options to pass to MATLAB."
```

Comment:
This help doesn't seem consistent with our other plugins. @mw-hrastega ?

Reply:
Agreed. We can use the same summary as in the run-build action.

Reply:
I think the best solution for this file is to use the help text from MATLAB actions.

Reply:
This is the help text in action.yml
https://github.com/matlab-actions/run-command/blob/main/action.yml